### PR TITLE
docs: update readme

### PR DESCRIPTION
**Pull request by archerny**
# Which issue does this PR close?

Closes #

# Rationale for this change
 
update readme, english version

# What changes are included in this PR?

update readme, english version

# Are there any user-facing changes?

no

# How does this change test

no need

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,188 @@
...
-Java client for CeresDB
+# CeresDB Java Client
+[中文](./docs/README_CN.md)
+
+## Introduction
+CeresDBxClient is a high-performance  Java client for CeresDB. CeresDB is a high-performance, distributed, schema-less, cloud native time-series database that can handle both time-series and analytics workloads.
+
+## Features
+- With the well designed SPI, the network transport layer is extensible. And we provide the default implementation which uses the gRPC framework.
+- The client provides high-performance async streaming write API
```
Comment (chunshao90) [inline README.md]: period

File: README.md [hunk 2]
```diff
@@ -1,2 +1,188 @@
...
+- The client also collects lots of performance metrics by default. These metrics can be configured to write to local file
```
Comment (chunshao90) [inline README.md]: period

File: README.md [hunk 3]
```diff
@@ -1,2 +1,188 @@
-# ceresdb-java-client
-Java client for CeresDB
+# CeresDB Java Client
+[中文](./docs/README_CN.md)
+
+## Introduction
+CeresDBxClient is a high-performance  Java client for CeresDB. CeresDB is a high-performance, distributed, schema-less, cloud native time-series database that can handle both time-series and analytics workloads.
```
Comment (chunshao90) [inline README.md]: newline

File: README.md [hunk 4]
```diff
@@ -1,2 +1,188 @@
...
+- With the well designed SPI, the network transport layer is extensible. And we provide the default implementation which uses the gRPC framework.
+- The client provides high-performance async streaming write API
+- The client also collects lots of performance metrics by default. These metrics can be configured to write to local file
+- We can take memory snapshots that contains the status of critical objects. The snapshots can also be configured to write to local file, which helps a lot when we diagnose complex problems.
+
+## Data ingestion process
+
+```
+                   ┌─────────────────────┐  
+                   │   CeresDBxClient    │
```
Comment (chunshao90) [inline README.md:18]: ceresdbx -> ceresdb

File: README.md [hunk 5]
```diff
@@ -1,2 +1,188 @@
...
+    "c12 UINT32 NULL,"
+    "c13 UINT16 NULL,"
+    "c14 UINT8 NULL,"
+    "c15 TIMESTAMP NULL,"
+    "c16 VARBINARY NULL,"
+    "TIMESTAMP KEY(ts)) ENGINE=Analytic"
+);
+```
+
+For more examples of table creation statements, see the detail docs [here](./table.md)
```
Comment (chunshao90) [inline README.md]: * period
* docs url is 404

File: README.md [hunk 6]
```diff
@@ -1,2 +1,188 @@
...
+
+
+## Data ingestion example
+```java
+// CeresDBx options
+final CeresDBxOptions opts = CeresDBxOptions.newBuilder("127.0.0.1", 8081) //
```
Comment (chunshao90) [inline README.md:128]: CeresDBx -> CeresDB

File: README.md [hunk 7]
```diff
@@ -1,2 +1,188 @@
...
+        .tenant("test", "sub_test", "test_token") // tenant info
+        .writeMaxRetries(1) // maximum retry times when write fails (only some error codes will be retried, such as the routing table failure)
```
Comment (chunshao90) [inline README.md]: comment is too long, add a newline

File: docs/README_CN.md [hunk 8]
```diff
@@ -0,0 +1,187 @@
+# CeresDB Java Client
+
+## 介绍
+CeresDBxClient 是 CeresDB 的高性能 Java 版客户端。CeresDB 是定位为高性能的、分布式的、Schema-less 的云原生时序数据库。它可以同时支持时间序列和数据分析型的工作负载。
```
Comment (chunshao90) [inline docs/README_CN.md]: ceresdb

File: README.md (hunk 4 above)
Comment (archerny) [reply README.md:18]: Names in codes are still ceresdbx, so we keep the doc be consistent with codes.

File: README.md (hunk 6 above)
Comment (archerny) [reply README.md:128]: See the comments above, this will not be modified.

File: README.md [hunk 9]
```diff
@@ -1,2 +1,193 @@
-# ceresdb-java-client
-Java client for CeresDB
+# CeresDB Java Client
+[中文](./docs/README_CN.md)
+
+## Introduction
+CeresDBxClient is a high-performance  Java client for CeresDB.
```
Comment (ShiKaiWi) [inline README.md]: Remove the unnecessary whitespace.

File: README.md [hunk 10]
```diff
@@ -1,2 +1,193 @@
...
+    "c13 UINT16 NULL,"
+    "c14 UINT8 NULL,"
+    "c15 TIMESTAMP NULL,"
+    "c16 VARBINARY NULL,"
+    "TIMESTAMP KEY(ts)) ENGINE=Analytic"
+);
+```
+
+For more examples of table creation statements, see the detail docs [here](./docs/table.md)
+
```
Comment (ShiKaiWi) [inline README.md]: Remove this line?

File: README.md [hunk 11]
```diff
@@ -1,2 +1,193 @@
...
+                    └─────────────────────┘  
+                               ▲  
+                               │  
+                               ▼  
+                    ┌─────────────────────┐  
+                    │  Default gRPC impl  │  
+                    └─────────────────────┘  
+                               ▲  
+                               │  
+           ┌───────────────────┴ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ┐
```
Comment (ShiKaiWi) [inline README.md:46]: Any differences between the solid line and the dotted line?

File: README.md [hunk 12]
```diff
@@ -1,2 +1,193 @@
...
+final Result<QueryOk, Err> qr = qf.get();
+
+Assert.assertTrue(qr.isOk());
+
+final QueryOk queryOk = qr.getOk();
+
+final List<Record> records = queryOk.mapToRecord().collect(Collectors.toList())
+```
+## Licensing
+CeresDB and its Java client is under [Apache License 2.0](./LICENSE).
```
Comment (ShiKaiWi) [inline README.md]: Maybe just `Under [Apache License 2.0](./LICENSE).` is enough.

File: README.md (hunk 11 above)
Comment (archerny) [reply README.md:46]: The dotted lines represent many CeresDB instances.

File: README_CN.md [hunk 13]
```diff
@@ -176,5 +176,10 @@ final QueryOk queryOk = qr.getOk();
 final List<Record> records = queryOk.mapToRecord().collect(Collectors.toList())
 ```
 
-## 技术支持
-- 顾客是上帝钉钉群：21983792
+## Licensing
+CeresDB 与其 Java client 都遵守 [Apache License 2.0](./LICENSE).
+
+## 社区与技术支持
+- 加入 Slack 社区与用户群：[Slack 入口](https://join.slack.com/t/ceresdbcommunity/shared_invite/zt-1au1ihbdy-5huC9J9s2462yBMIWmerTw)
```
Comment (ShiKaiWi) [inline README_CN.md]: Maybe it is better to remove the `：`

File: README_CN.md [hunk 14]
```diff
@@ -176,5 +176,10 @@ final QueryOk queryOk = qr.getOk();
...
+- 加入微信社区与用户群 [微信二维码](https://github.com/CeresDB/assets/blob/main/WeChatQRCode.jpg)
+- 搜索并加入钉钉社区与用户群: 44602802
```
Comment (ShiKaiWi) [inline README_CN.md]: Maybe it is better to remove the `：`

**Review by ShiKaiWi — APPROVED** [review]
